lib_stusb/STM32_USB_Device_Library/Class/CCID/inc/usbd_ccid_if.h

```diff
@@ -223,6 +223,8 @@ void Set_CSW (uint8_t CSW_Status, uint8_t Send_Permission);
 
 void io_usb_ccid_set_card_inserted(unsigned int inserted);
 
+void io_usb_ccid_activate_pinpad(uint8_t enabled);

```

Comment:
There is no need for a `desactivate` function?
Does this means we could always enable the feature and make this a no-op?

Reply:
Cause this will be somehow what will happen for next app launch.

Reply:
Ok my bad, I didn't pay attention to the parameter...
Could you rename it with configure instead of activate?

Reply:
Done